Tests/kaas/k8s-node-distribution/k8s-node-distribution-check.py

```diff
@@ -99,6 +79,7 @@ def print_usage():
 The following arguments can be set:
     -c/--config PATH/TO/CONFIG         - Path to the config file of the test script
     -k/--kubeconfig PATH/TO/KUBECONFIG - Path to the kubeconfig of the server we want to check
+    -t/--test PATH/TO/YAML             - Input a formatted yaml file to test the script functionality
```

Comment:
This is a good approach, but see my other comment about splitting the main function for easier testing on the unit testing level.

With this `--test` parameter, you would need to test the script as a whole, e.g., via `subprocess.run()` and check the stdout/stderr output.

Reply:
Did some changes here https://github.com/SovereignCloudStack/standards/pull/558/commits/aec75ef610ce03dc848181b9a85fefc6354b6ed0